build.ps1

```diff
@@ -1,6 +1,10 @@
 # Copyright 2020 Saleem Abdulrasool <compnerd@compnerd.org>
 # Copyright 2023 Tristan Labelle <tristan@thebrowser.company>
 
+param(
+  [string[]] $Archs = @("X64","X86","Arm64")
```

Comment:
Please name this `SDKs`.  It is ambiguous for the architecture for the toolchain vs SDK.  Thoughts on `ARM64` rather than `Arm64`?

Reply:
Done in latest. String comparisons are case insensitive in Powershell, even the switch statement, I verified